pkg/common/util/sort.go

```diff
@@ -140,6 +149,20 @@ func compareTypesEntries(a, b *types.Entry) int {
 		return 1
 	}
 
+	switch {
+	case a.X509SvidTtl < b.X509SvidTtl:
+		return -1
+	case a.X509SvidTtl > b.X509SvidTtl:
+		return 1
+	}
+
+	switch {
+	case a.JwtSvidTtl < b.JwtSvidTtl:
+		return -1
+	case a.JwtSvidTtl > b.JwtSvidTtl:
+		return 1
+	}
+
```

Comment:
This structure differs from a very similar change above (https://github.com/spiffe/spire/pull/3445/files?diff=split&w=1#diff-d6a62203136add25829a896664299adabed15dabe8ab4cc857cf9570a040a75dR63-R75) where I accomplish the same thing in a single `switch` clause. Sloppy on my part, but before changing one or the other I wanted to get opinions on which is preferred.